src/Planner/findParallelReplicasQuery.cpp

```diff
@@ -401,6 +401,11 @@ static const TableNode * findTableForParallelReplicas(const IQueryTreeNode * que
                 query_tree_node = array_join_node.getTableExpression().get();
                 break;
             }
+            case QueryTreeNodeType::CROSS_JOIN:
+            {
+                /// TODO: We can parallelize one table
```

Comment:
Forgot, - let's add test with CROSS JOIN in subquery with PR to cover this lines